packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -173,7 +175,7 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
         shouldStartVisible={imageShouldStartVisible}
         onLoadingStateChange={this._onPhotoLoadingStateChange}
       />
-    );
+    ) : null;
```

Comment:
>null [](start = 8, length = 4)

We can move the null imageUrl check to the top of the function and return null immediately in that case